airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -135,6 +135,279 @@ definitions:
       $ref: "#/definitions/retriever_without_pagination"
 
   # Stream Definitions
+  companies_property_history_stream:
+    $ref: "#/definitions/stream_base"
+    primary_key:
+      - companyId
+      - property
+      - timestamp
+    retriever:
+      $ref: "#/definitions/base_retriever"
+      requester:
+        $ref: "#/definitions/base_requester"
+        request_parameters:
+          propertyMode: value_and_history
+          propertiesWithHistory:
```

Comment:
noting this because anatolii and I discussed. The existing connector had a request parameter in the format: `&propertiesWithHistory=associatedcompanyid&propertiesWithHistory=hs_email_hard_bounce_reason`.

However, now with our property chunking we emit as `&propertiesWithHistory=associatedcompanyid,hs_email_hard_bounce_reason` with just comma delimiting. I tested this on the v3 endpoints and it appears to work as expected